src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+  .calendar {
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: calc(7 * $calendar-day-width + 7px);
```

Comment:
The '7px' addition in the width calculation seems arbitrary. If this is meant to account for margins or borders, it should be calculated based on a SCSS variable or a well-documented constant to improve code maintainability and clarity.